internal/codespaces/rpc/invoker.go

```diff
@@ -116,9 +116,8 @@ func connect(ctx context.Context, fwd portforwarder.PortForwarder) (Invoker, err
 		// Attempt to connect to the port
 		opts := []grpc.DialOption{
 			grpc.WithTransportCredentials(insecure.NewCredentials()),
-			grpc.WithBlock(),
 		}
-		conn, err = grpc.DialContext(connectctx, localAddress, opts...)
+		conn, err = grpc.NewClient(localAddress, opts...)
```

Comment:
Looks like a legit change as described by:
 * https://github.com/grpc/grpc-go/issues/7244
 * https://github.com/grpc/grpc-go/blob/master/Documentation/anti-patterns.md#how-to-properly-create-a-clientconn-grpcnewclient

Reply:
Best I can tell the commands that use this still work:

```
➜  sigstore-go git:(main) ~/workspace/cli/bin/gh cs ssh
? Choose codespace: github/github (wm/remove-device_authorization_grant_replica_race): ubiquitous capybara
Welcome to Ubuntu 20.04.6 LTS (GNU/Linux 6.5.0-1022-azure x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/pro
Last login: Thu Jul 25 05:34:25 2024 from ::1
```

```
➜  sigstore-go git:(main) ~/workspace/cli/bin/gh cs jupyter
? Choose codespace: ipdor/StudyAtCoursera (master): organic space enigma
http://localhost:53733/?token=27038fd3fb270ee7e4fd4d6710c5d21de389c03539091fd8
```